src/components/auth/socialauth/__tests__/socialAuth.test.js

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable comma-dangle */
+import React from "react";
+
+import Enzyme, { shallow, mount } from "enzyme";
+import { MemoryRouter } from "react-router-dom";
+import Adapter from "enzyme-adapter-react-16";
+import { Provider } from "react-redux";
+import configureStore from "redux-mock-store";
+import thunk from "redux-thunk";
+import jest from "jest-mock";
+import SocialLoginComponent, { mapDispatchToProps } from "../SocialLogin";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+const middlewares = [thunk];
+const fakeStore = configureStore(middlewares);
+
+describe("Login Component snapshot", () => {
+  it("should render the Login component as expected", () => {
+    const component = shallow(<SocialLoginComponent debug />);
+    expect(component).toMatchSnapshot();
+  });
+});
+
+describe("render the component and check if all items exist", () => {
+  const store = fakeStore({
+    socialAuth: {
+      name: "Charity",
+      message: "fetching"
+    }
+  });
+  const wrapper = mount(
```

Comment:
Nice wrapper here to check if all the components render, I get insight on how to test my feature